subprojects/core/src/main/java/org/gradle/api/internal/tasks/properties/TaskProperties.java

```diff
@@ -57,6 +57,7 @@
      */
     FileCollection getInputFiles();
 
+    // TODO Can we remove this now?
```

Comment:
In my PR (https://github.com/gradle/gradle/pull/14983) I managed to remove a number of these properties, let's make sure that we do it here, too.

Reply:
I removed the same things as you did in your PR now.